media/js/mozorg/contact-spaces.js

```diff
@@ -39,6 +39,8 @@
             var $main = $('#main-content');
             // get the mapbox map id.
             var mapId = $main.data('mapbox');
+            // touch support detection.
+            var hasTouch = L.Browser.touch || L.Browser.msTouch;
```

Comment:
Would this be better as a higher level variable so as to not declare/assign twice?